env.common.yml

```diff
@@ -29,3 +29,5 @@ dependencies:
     - transformers==4.6.*
     - xgboost==1.*
     - git+https://github.com/samoturk/mol2vec
+    - git+https://github.com/deepmind/dm-haiku
```

Comment:
Can you add these dependencies to the soft dependencies list in the docs in `docs/source/get_started/requirements.rst`?

Reply:
Is it really necessary to add these dependencies?  Our dependency matrix is already reaching a point where installing DeepChem can be impossible.  We've had to put off our next release because of it.  Every extra dependency makes the problem even worse.  We desperately need to be getting rid of dependencies, not adding new ones.

Reply:
Perhaps we should add a `deepchem-jax` package as we've previously discussed instead of adding on to the dependencies of the base package. It's definitely true that the dependency matrix for deepchem has started to become unsustainable.

Reply:
+1 for a `deepchem-jax` package that doesn't include PyTorch, Tensorflow, dgl, transformers, etc. Would vastly simplify the dependencies here.